SimPEG/DataMisfit.py

```diff
@@ -58,19 +58,20 @@ class l2_DataMisfit(BaseDataMisfit):
 
     """
 
-    std = 0.05  #: default standard deviation if not provided by survey
+    std = None  #: default standard deviation if not provided by survey
     eps = None  #: default floor
     eps_factor = 1e-5  #: factor to multiply by the norm of the data to create floor
 
     def __init__(self, survey, **kwargs):
         BaseDataMisfit.__init__(self, survey, **kwargs)
 
         if self.std is None:
-            if getattr(self.survey, 'std', None) is not None:
+            if getattr(self.survey, 'std', None) is None:
```

Comment:
Thanks for catching this @thast. This won't yet catch the case if the user changes the standard deviation on the survey after the data misfit has been created. I think that is okay for now, but lets leave an issue open and we can work to address this with the simulation refactor.